andrepereiradasilva/joomla-cms
# [com_search] Implement searchtools

Pull Request for Improvement.
#### Summary of Changes

This PR, like already done in most views across Joomla admin panel, implements searchtools in com_search searches view.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/14703168/6438003c-07a6-11e6-8466-3bd18a8caa6c.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/14703139/32e3cf66-07a6-11e6-9d22-2d85c71e58e7.png)
#### Testing Instructions
1. Apply this patch in latest staging
2. Go to Components -> Search
3. Go to Options and enable gathering statistics
4. Go to the frontend `index.php?option=com_search&view=search` URI and do some searches.
5. Go again to Components -> Search, you'll see some search results
6. Apply this patch
7. Go again to Components -> Search you'll see the new visual
8. Test if search, filters, ordering and page limit (aka searchtools) are working properly. Test also the hide/show search results button.
